### PR TITLE
Precision float view

### PR DESCRIPTION
**Pull request by KateMengjunXia**
Part of https://github.com/nanoPayinc/NANOPAY/pull/6311

### CODE REVIEW

File: src/foam/u2/PrecisionFloatView.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            if ( text.indexOf('.') > 0 && text.length - text.indexOf('.') - 1 > this.precision ) {
+              this.attrSlot(null, this.onKey ? 'input' : null).set(text.substring(0, text.indexOf('.') + this.precision + 1));
+            }
+          }.bind(this));
+        }
+      }
+    ]
+  });
+    
+
```
Comment (zahnno) [inline src/foam/u2/PrecisionFloatView.js]: remove extra empty space

**Review by zahnno — APPROVED** [review]
LG have one minor comment

File: src/foam/u2/PrecisionFloatView.js (hunk 1 above)
Comment (KateMengjunXia) [reply src/foam/u2/PrecisionFloatView.js]: Removed

**Review by jlhughes — APPROVED** [review]

**Comment by kgrgreer** [comment]
This code has a bug. Please revert. Add functionality to the standard FloatView.

**Pull request merged**